cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
         )
         return data
```

Comment:
In the `validate` method of `TicketSerializer`, you're calling the `validate_ticket` method of the `Ticket` model. While this is not incorrect, it might be a good idea to move this validation logic into the serializer itself. Serializers in DRF are not only for transforming data but also for validation. It would make your code more idiomatic and easier to understand if all validation logic related to serialization is present in the serializers.